shell/components/ClusterBadge.vue

```diff
@@ -23,6 +23,7 @@ export default {
     :style="{ backgroundColor: cluster.badge.color, color: cluster.badge.textColor }"
     class="cluster-badge"
     :class="{'cluster-badge-border': showBorders}"
+    :aria-label="t('clusterBadge.badgeAppearance')"
```

Comment:
`Cluster Badge Appearance` doesn't quite describe what this is. It should probably be something like `Cluster comment: <actual text>`

Reply:
actually, this one contains text. will screen readers use the aria-label as a prefix for the text within the element?

Reply:
Probably (tbh, idk how screen readers will behave on this matter), but I definitely bet on having a complete `aria-label` completely address a11y.

I'm going with your `Cluster comment: <actual text>` approach. I had confused this component with the button one in cluster explorer main page. Thanks